src/main/java/uk/gov/hmcts/darts/event/mapper/EventEntityToCourtLogMapper.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.darts.event.mapper;
+
+import lombok.experimental.UtilityClass;
+import uk.gov.hmcts.darts.common.entity.EventEntity;
+import uk.gov.hmcts.darts.event.model.CourtLog;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@UtilityClass
+public class EventEntityToCourtLogMapper {
+
+    public List<CourtLog> mapFromEntityToCourtLogs(List<EventEntity> entities, String courthouse, String caseNumber) {
```

Comment:
probably don't need the `FromEntity` part of the name, as its implied by the fact that it accepts an EventEntity List parameter, so you could name this method `mapToCourtLogList`